challenges/dynamic_ball_handling_challenge.tex

```diff
@@ -1,3 +1,138 @@
+		\label{fig:ball_handling_arc_positions}
+	\end{center}
+\end{figure}
+
+\subsection{Challenge Scoring}
+\label{sec:Challenge_scoring}
+In each phase of a run the GameController measures the execution time from initial whistle until the run is stopped by one of the following criteria:
+
+\begin{itemize}
+	\item A goal is scored after at least two passes out of this list have been executed:
+	\begin{itemize}
+		\item valid pass,
+		\item semi-valid pass,
+		\item substantial pass attempt.
+	\end{itemize}
+	\item A defender touches the ball.
+	\item Ball leaves the field outside the defending goal area.
+	\item An attacker pushes.
+	\item There is only one attacker left on the field.
+	\item A run exceeds \qty{4}{\minute} execution time.
+	\item Attacking team is not communicating with the GameController.
+\end{itemize}
+
+In the case that a goal gets scored before two passes (see list above) were executed, or the ball leaves the field within the defending goal area, the ball gets placed on the closest goal kick spot.
+
+The score for the attacking team in a run will be calculated based on the following rules:
+
+\begin{enumerate}
+	\item For the first two passes, add 
+	\begin{enumerate}
+		\item \textbf{valid pass}: \qty{30}{} points.
+		\item \textbf{semi-valid pass}: \qty{20}{} points.
+		\item \textbf{substantial pass attempt}: \qty{10}{} points.
+	\end{enumerate}
+	\item For a third pass, add
+	\begin{enumerate}
+		\item \textbf{valid pass}: \qty{20}{} points.
+		\item \textbf{semi-valid pass}: \qty{13}{} points.
+		\item \textbf{substantial pass attempt}: \qty{6}{} points.
+	\end{enumerate}
+	\item For a fourth pass, add
+	\begin{enumerate}
+		\item \textbf{valid pass}: \qty{10}{} points.
+		\item \textbf{semi-valid pass}: \qty{6}{} points.
+		\item \textbf{substantial pass attempt}: \qty{3}{} points.
+	\end{enumerate}
+	\item For a goal after at least two passes, add \qty{50}{} points.
+	\item The time measured counts and can lead to \qty{120}{} points if \qty{0}{\second} are needed and \qty{0}{} points if \qty{240}{\second} are needed. In between full points (rounded) will be awarded linearly. \\
+	\textit{However these points are not awarded if the run was stopped by a stopping criteria different to scoring a goal, see above.}
+	\item If an attacking robot has been pushed by a defender, subtract \qty{20}{\second} of the time measured.
+\end{enumerate}
+
+The final score is the mean time out of three runs.
+
+\subsection{Defender image}
+\label{sec:Challenge_image}
+Common defender images can be provided by the community with a standardized setup procedure and with automatic calibration. Every team can propose such an image until 2023-03-12. The image will be tested until 2023-03-31 if they match the requirements and afterwards published. If not, the team gets feedback and has the opportunity to hand in a revised image. Defender images from last year can be used again.
```

Comment:
> Defender images from last year can be used again.
But they will likely not be compatible with the GameController for this year. It's not *impossible* to have the GameController send "old" packets in the mode for this challenge, too, but I would prefer to avoid this.

Reply:
Resolved in fe7e25b